tools/db_bench_tool.cc

```diff
@@ -790,6 +790,8 @@ DEFINE_string(compression_type, "snappy",
 static enum rocksdb::CompressionType FLAGS_compression_type_e =
     rocksdb::kSnappyCompression;
 
+DEFINE_int64(sample_for_compression, 0, "Sample every N block for compression");
```

Comment:
Since we support setting `sample_for_compression` via command line when running db_bench, could you update the PR description to include an updated test plan with instructions in running db_bench, and show the result of the program execution?

Reply:
Probably vary the value of `sample_for_compression` to give people an idea of possible overhead.

Reply:
Updated the PR.